showcase/src/app/welcome/pages/home/_home-page.component.scss

```diff
@@ -1,94 +1,168 @@
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
 }
 
-.bottom-margin {
-  margin-bottom: 20px;
+.about-image {
+  mask-image: url("/assets/images/logo/wall.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($primary-palette, 500);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
 }
 
-//icon with a link to github
-.github-link .github-icon {
-  height: 60px;
-  width: 60px;
-  position: relative;
-  float: right;
+.ui-image {
+  mask-image: url("/assets/images/logo/television-guide.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($success-palette, 500);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
+}
 
-  svg {
-    fill: #000;
-  }
+.showcase-image {
+  mask-image: url("/assets/images/logo/theater.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  background-color: mat-color($warning-palette, 900);
+  height: 80px;
+  width: 80px;
+  min-width: 80px;
+  max-width: 80px;
 }
 
-.information-title {
-  font-size: 21px;
-  color: mat-color($grey-palette, 900);
-  padding-bottom: 0;
-  display: inline-flex;
-  vertical-align: middle;
-  margin: 10px 0 10px 0;
+.page-content {
+  margin: 0;
 }
 
-@media #{$desktop-query} {
-  .about-title {
-    font-size: 55px;
-    margin-left: 25px;
-  }
+.doc-description {
+  display: flex;
+  margin-left: 25px;
+  margin-right: 25px;
 
-  .information-title {
-    font-size: 30px;
+  p {
+	display: flex;
   }
+}
 
-  .about-background {
-    height: 150px;
-    background-size: 250px 75px;
-  }
+.custom-button {
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  text-decoration: none;
+  height: 40px;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 40px;
+  color: #fff;
+  background-color: #0063bb;
+  border-radius: 48px;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, .26);
+  box-sizing: border-box;
+  cursor: pointer;
+  display: block;
+  margin: auto;
+  padding: 0 24px;
+  text-align: center;
+}
+
+.custom-buttons {
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  text-decoration: none;
+  height: 40px;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 40px;
+  color: #fff;
+  background-color: #0063bb;
+  border-radius: 48px;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, .26);
+  box-sizing: border-box;
+  cursor: pointer;
+  display: inline-flex;
+  margin: auto 10px auto 10px;
+}
+
+.buttons-row {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.section-division {
+  border-bottom: 1px solid #bdbdbd;
+  padding-bottom: 20px;
+  padding-top: 20px;
 }
 
-@media #{$tablet-only-query} {
-  .list {
-    font-size: 17px;
+@media #{$desktop-lg-query},#{$tablet-only-query},#{$desktop-query}{
+
+  .button.custom-buttons {
+	margin: auto 30px auto 30px;
   }
 
-  .about-background {
-    height: 100px;
-    background-size: 250px 75px;
+  .core-image, .about-image, .ui-image, .showcase-image {
```

Comment:
Can you merge these into one single class and apply it to the 3 images? To clean a bit the CSS ;)